src/main/flight/pid.c

```diff
@@ -272,6 +283,8 @@ void pidInitFilters(const pidProfile_t *pidProfile)
     const uint32_t pidFrequencyNyquist = pidFrequency / 2; // No rounding needed
 
     r_weight = (float) pidProfile->r_weight / 100.0f;
+	errorMultiplier = (float)pidProfile->errorBoost * (float)pidProfile->errorBoost * 1e-9f;
```

Comment:
If we change this line to be 

((float)pidProfile->errorBoost * (float)pidProfile->errorBoost * 1e-9f) * 0.003

then the emuboost will retain the same scaling as before.

Reply:
div 1000000 * .003 is how much?
1e-9 i think - six plus three would be nine.

Reply:
I just did that math and got 3e-9
so if we change that 1e-9 everything should be right.